### DISCUSSION TITLE
Look likes `unplugin-parcel-macros` is not working with `nextjs 15`?

### DISCUSSION THREAD

**Discussion by ladifire** [adobe/react-spectrum#7527] [General]
Currently I've setup a Nextjs 15 with `unplugin-parcel-macros` but look likes it's not working. Do we have any idea on this?

**Comment by snowystinger** [comment]
We have an example using 14 https://github.com/adobe/react-spectrum/tree/main/examples/s2-next-macros
Are there any noticeable differences?

Would you be willing to put together an example app showing it? Could make it in a PR to match the v14 one.
